src/makeCalculator.js

```diff
@@ -37,7 +37,42 @@
  * @return {object}
  */
 function makeCalculator() {
-  // write code here
+  const calculator = {
+    result: 0,
+
+    add: value => {
+      calculator.result += value;
+
+      return calculator;
+    },
```

Comment:
make it no-arrow function and use `this` instead of `calculator`

Reply:
I wrote like that, but it doesn`t work
https://drive.google.com/file/d/1Vem52sPst6wgqpShVPphntuSvegl7nXS/view?usp=sharing